Document pre-commit git hook usage (#200)
returntocorp/bento: README.md

  <span> · </span>
  <a href="#running-bento-in-ci">CI/CD</a>
  <span> · </span>

>   <a href="#running-bento-as-a-git-hook">Git Hook</a>

Reviewer: An idea to slightly simplify the top menu would be to replace CI/CD and Git Hook with “Automation” and then create an Automation section below at the same level (`##`) as Installation, Usage, Help & Community. Automation would have two subsections, CI/CD and Git Hook. What do you think?

Otherwise, you can simply install and run Bento in CI with the following commands:

```bash

> $ pip3 install bento-cli && bento --version

Reviewer: Curious, why is the `bento --version` needed here?

Author: Not needed, but it ensures in your log you know what version of Bento ran

Reviewer: 👍

$ bento --agree --email <YOUR_EMAIL> check
```

`bento check` will exit with a non-zero exit code if it finds issues in your code (see [Exit Codes](#exit-codes)). To suppress this behaviour you can pipe its output to `true`:

 ```bash
 $ bento --agree --email <YOUR_EMAIL> check || true
 ```
 
Otherwise, address the issues or archive them with `bento archive`.

If you need help setting up Bento with another CI provider please [open an issue](https://github.com/returntocorp/bento/issues/new?template=feature_request.md). Documentation PRs welcome if you set up Bento with a CI provider that isn't documented here!

### Running Bento as a Git Hook
Bento can automatically analyze your staged files when `git commit` is run. Configured as a Git pre-commit hook, Bento ensures every commit to your project is vetted and that no new issues have been introduced to the codebase.

> To install Bento as a a Git hook:

Reviewer: ```suggestion
To install Bento as a Git hook:
```

```bash
$ bento install-hook
```

> If Git hooks ever incorrectly block your commit you can always skip them by passing the `--no-verify` flag at commit-time (use this sparingly):

Reviewer: ```suggestion
If Git hooks ever incorrectly block your commit, you can skip them by passing the `--no-verify` flag at commit-time (use this sparingly):
```

```bash
$ git commit --no-verify
```

> Bento's Git hook can save the round-trip time involved with fixing a failed build if you're using [Bento in CI](#running-bento-in-ci). 

Reviewer: ```suggestion
Bento’s Git hook can save the round-trip time involved with fixing a failed build if you’re using [Bento in CI](#running-bento-in-ci). 
```
